### ISSUE TITLE
Create default tags

### ISSUE THREAD

**Issue by chriscasto** [metrotranscom/doorway-infra#3]
Depends on #2

Define a set of tags to be applied to any resources created so resources within the same account can be differentiated.  These are the ones we should start with:

Team
Project
Application
Environment (ie stage in the software development lifecycle)

Note that all of these tags correspond to variables created in a separate issue.

**Comment by chriscasto** [comment]
In the TF demo we did previously, tags were defined in a local and then explicitly passed into each resource (and modified as needed).  The AWS provider has a [default_tags](https://developer.hashicorp.com/terraform/tutorials/aws/aws-default-tags) arg where these can be set in one place and applied automatically to all resources.  Our default tags should be defined here and then referenced elsewhere as needed using `data.aws_default_tags.current.tags`.
